### ISSUE TITLE
low priority: optional to have summary.makesense method

### ISSUE THREAD

**Issue by mikejacktzen** [statsccpr/sensemakr#12]
low priority: optional to have summary.makesense method

would be useful if summary() has quantities that print() does not

**Comment by chadhazlett** [comment]
Agreed. At minimum, it would be good if summary produced all the alphas and gammas for each covariate.
